reference/6/Microsoft.PowerShell.Core/About/about_Environment_Variables.md

```diff
@@ -241,13 +241,19 @@ The environment variables that store preferences include:
   for modules in the specified directories when you do not specify a full path
   to a module.
 
-  The default value of $Env:PSModulePath is:
+  The default value of $Env:PSModulePath on Windows is:
 
   ```
   $HOME\Documents\WindowsPowerShell\Modules; $PSHOME\Modules
   ```
 
-PowerShell sets the value of "\$PSHOME\\Modules" in the registry. It
+  The default value of $Env:PSModulePath on Linux or MacOS is:
+
+  ```
+  $HOME/.local/share/powershell/Modules:/usr/local/share/powershell/Modules:/opt/microsoft/powershell/6/Modules
```

Comment:
The `/opt/...` path is Linux-specific; on macOS, it's different. Also, it's better to avoid specific version numbers in the paths.